pkg/controller/status/controller.go

```diff
@@ -95,6 +95,9 @@ func StartKubeFedStatusController(controllerConfig *util.ControllerConfig, stopC
 func newKubeFedStatusController(controllerConfig *util.ControllerConfig, typeConfig typeconfig.Interface) (*KubeFedStatusController, error) {
 	federatedAPIResource := typeConfig.GetFederatedType()
 	statusAPIResource := typeConfig.GetStatusType()
+	if statusAPIResource == nil {
+		return nil, fmt.Errorf("spec.statusType is nil")
```

Comment:
-> `"Status collection is not supported for %q", federatedAPIResource.Kind`

Reply:
Also, as per the CI failure, please prefer use of the [errors](https://dave.cheney.net/2016/06/12/stack-traces-and-the-errors-package) package to ensure traceability (e.g. `errors.Errorf`).

Reply:
done, thanks!